DOCKER.md

```
@@ -76,6 +77,35 @@ cp docker/callofcode.in/.env.local.frontend.example docker/callofcode.in/.env.lo
+
+| Service | Container | Host port | Override env var |
+|---------|-----------|-----------|------------------|
+| `coc-api` | `3000` | **3000** | `COC_API_PORT` |
+| `callofcode-frontend` | `3001` | **3001** | `CALLOFCODE_PORT` |
+| `coc-admin-server` | `8000` | **8001** | `COC_ADMIN_BACKEND_PORT` |
+| `coc-admin-web` | `5173` | **5174** | `COC_ADMIN_FRONTEND_PORT` |
+| `coc-member-server` | `8000` | **8002** | `COC_MEMBER_BACKEND_PORT` |
+| `coc-member-web` | `5173` | **5175** | `COC_MEMBER_FRONTEND_PORT` |
+
+> **Port remapping**: because `coc-admin` and `coc-member` both internally bind `8000` and `5173`, the master stack remaps them to unique host ports to avoid conflicts. Internal service-to-service communication (`API_URL=http://coc-api:3000`) still uses the original container ports.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)